mti_nma/steps/compare_nuc_cell/compare_nuc_cell.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+from pathlib import Path
+from typing import Dict, List, Optional, Union
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sb
+from datastep import Step, log_run_params
+
+# Run matplotlib in the background
+matplotlib.use('Agg')
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class CompareNucCell(Step):
+    def __init__(
+        self,
+        direct_upstream_tasks: List["Step"] = [],
+        config: Optional[Union[str, Path, Dict[str, str]]] = None,
+        filepath_columns: List[str] = [
+            "compare_hist_FilePath",
+            "compare_kde_FilePath",
+        ]
+    ):
+        super().__init__(
+            direct_upstream_tasks=direct_upstream_tasks,
+            config=config,
+            filepath_columns=filepath_columns,
+        )
+
+    @log_run_params
+    def run(self, nma_nuc_df, nma_cell_df, **kwargs):
+        """
+        Run a pure function.
+
+        Protected Parameters
+        --------------------
+        distributed_executor_address: Optional[str]
+            An optional executor address to pass to some computation engine.
+        clean: bool
+            Should the local staging directory be cleaned prior to this run.
+            Default: False (Do not clean)
+        debug: bool
+            A debug flag for the developer to use to manipulate how much data runs,
+            how it is processed, etc.
+            Default: False (Do not debug)
```

Comment:
I should have told you this so my bad, but these should be folded into the `Parameters` section of each docstring where needed. They are just always passed by default so if you don't use them then get rid of them but if you do put them in the primary `Parameters` sections.